pro-plan-for-free.js

```diff
@@ -10,5 +10,7 @@ module.exports = [
   'moonsmile', // 2019-9-1: Student writing tools to learn, I promise i will pay when I finish my project.
   'apache', // 2020-01-01: non-profit corporation to support Apache software projects
   'acts-project', // 2020-03-30: Generic track reconstruction framework for high energy physics
-  'caremessageplatform' // 2020-11-17: non-profit org, see https://git.io/Jk8Uw
+  'caremessageplatform', // 2020-11-17: non-profit org, see https://git.io/Jk8Uw
+  'AraHaan', // 2021-04-23: Open source developer who belives that software and libraries should be open source. This is so they can continue to be maintained even by the community. Everything I do is mostly open source, free, and for learning purposes as well. Developer who learns without schooling currently.
+  'Elskom' // 2021-04-23: Made by me (@AraHaan on github) to move open source projects that was directly on my user account to an org so that way I can easily add contributors for globally with ease. Everything made is always free and open source and I plan to eventually set up donations for if the users ever plan to donate to support development of the libraries/software made there when I figure it out 😭. I will try to buy the pro plan if and when I can figure out the donations part and when I start getting donations.
```

Comment:
no need for such long explanations :) This is code that is used in production, I'd like to keep it short. You can just say that you are a student and would like to use WIP in your Open Source projects, and for @Elskom that it's an org account of open source projects by @AraHaan with outside collaborators

Reply:
Alright, updated.